demos/shadow/shadow_demo_main/shadow_demo_main.c

```diff
@@ -774,13 +780,17 @@ int main( int argc,
              * uint16_t outLength = 0;
              * const char * thingName = "TestThingName";
              * uint16_t thingNameLength  = ( sizeof( thingName ) - 1U );
+             * const char * shadowName = "TestShadowName";
+             * uint16_t shadowNameLength  = ( sizeof( shadowName ) - 1U );
```

Comment:
please refer to the fix in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1527 and update accordingly.

Reply:
I think I must have messed a merge up because I fixed this previously. Anyway, done.